dev-libs/libpipeline/libpipeline-1.5.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A C library for setting up and running pipelines"
+DESCRIPTION="libpipeline is a C library for setting up and running pipelines \
+of processes, without needing to involve shell command-line parsing which is \
+often error-prone and insecure.  This alleviates programmers of the need to \
+laboriously construct pipelines using lower-level primitives such as fork(2) \
+and execve(2)."
+HOMEPAGE="http://libpipeline.nongnu.org/"
+COPYRIGHT="1994 Markus Armbruster.
+	1989, 1990, 1991, 1992, 2000, 2001, 2002, 2003, 2004, 2005,
+	2006, 2007, 2008, 2009, 2010 Free Software Foundation, Inc.
+	2003-2017 Colin Watson"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://download.savannah.nongnu.org/releases/libpipeline/libpipeline-$portVersion.tar.gz"
+CHECKSUM_SHA256="0d72e12e4f2afff67fd7b9df0a24d7ba42b5a7c9211ac5b3dcccc5cd8b286f2b"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libpipeline$secondaryArchSuffix = $portVersion
+	lib:libpipeline$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libpipeline${secondaryArchSuffix}_devel = $portVersion
+	devel:libpipeline$secondaryArchSuffix = $portVersion
```

Comment:
```diff
 PROVIDES_devel="
 	libpipeline${secondaryArchSuffix}_devel = $portVersion
-	devel:libpipeline$secondaryArchSuffix = $portVersion
+	devel:libpipeline$secondaryArchSuffix = $libVersionCompat
	"
```

Reply:
**Important:** I don't know if it is appropriate to have a compat for `devel:lib*` lines.
Well, even if it were only appropriate for `lib:lib*` lines it would still be nice, imo, to define libVersionCompat right after libVersion.

Reply:
libVersionCompat isn't used at the moment by any other recipe?

Reply:
I recently added lib**V**ersion to openssl and later another contributor added lib**v**ersion to another recipe.
But for libVersionCompat, libpipeline would be the first recipe to have it ;-)
So maybe we should wait until some key contributors say if they agree with these new custom variables.

Reply:
Ok, added it, will wait for replies :)